packages/common/src/projects/_internal/ProjectBusinessRules.ts

```diff
@@ -36,7 +36,7 @@ export const ProjectCapabilityPermissions: ICapabilityPermission[] = [
   {
     entity: "project",
     capability: "settings",
-    permissions: ["hub:project:owner"],
+    permissions: ["hub:project:owner", "hub:workspace:released"],
```

Comment:
Wrapping it up in a project capability.

So, we can use `checkCapability('settings', context, projectEntity)`.

Or, in the context of hub components WorkspaceLinkDefinitions, we would do something like:
```js
{
    entity: 'project',
    pane: 'settings',
    component: 'arcgis-hub-entity-settings',
    i18nLabel: 'panes.settings',
    icon: PANE_ICONS.settings,
    capabilities: ['settings'],
  }
```

Finally, we can do `checkPermission('hub:workspace:released', context)`.

Reply:
using `temp:` of `flag:` would make this easier to reason about what will be removed down the line...